### DISCUSSION TITLE
Set Favorite HW TX and RX for updates

### DISCUSSION THREAD

**Discussion by Astor87** [ExpressLRS/ExpressLRS-Configurator#614] [Ideas]
Your configurator is really more that great!!!
 However just 1 more foolproof (at least for me) would be superb:
Provide Selection of favorites i.e. mark, select, group of  TX and RX  used  by  the individual user for updates ,  changes would/could eliminate flashing wrong updates. I did realize this when flashing  by mistake a BETAFPV 2.4G Nano V2 with the BETAFPV 2.4G Nano  ELRS 3.4.0 CE LBT Firmware. Did not check V2, did not work! Took me some time to find the error, because update with the configurator was reported successful! 
Would be a great foolproof improvement, since amount of hardware will increase !
Just an idea . Let ne know if you  are interested and need more explanation. Sorry English is not my native. 
BR 
Wolfram

**Comment by jurgelenas** [comment]
Hey @Astor87 

I have converted your GitHub issue to discussion thread.

Thanks for the idea.
